src/screens/EventDetailsScreen/component.js

```diff
@@ -0,0 +1,232 @@
+      ) && (
+        <View style={styles.iconItemWrapper}>
+          <IconItem
+            icon={<Text type="xSmall">icn</Text>}
+            title={text.eventDetailsGenderNeutralToilets}
+          />
+        </View>
+      )}
+      {event.fields.accessibilityOptions &&
+        event.fields.accessibilityOptions[locale].length > 0 && (
+          <View style={styles.iconItemWrapper}>
+            <IconItem
+              icon={<Text type="xSmall">icn</Text>}
+              title={text.eventDetailsAccessibility}
+              content={
+                <Text type="small">
+                  {event.fields.accessibilityOptions[locale].join(", ")}
+                </Text>
+              }
+            />
+          </View>
+        )}
+    </View>
+  );
+};
+
+const renderEventDescription = event => (
+  <View style={styles.content}>
+    <Text markdown>{event.fields.eventDescription[locale]}</Text>
+    <View style={styles.mapWrapper}>
+      <EventMap
+        lat={event.fields.location[locale].lat}
+        lon={event.fields.location[locale].lon}
+        locationName={event.fields.locationName[locale]}
+      />
+    </View>
+  </View>
+);
+
+const renderEventDetails = event =>
+  (event.fields.accessibilityDetails ||
+    event.fields.email ||
+    event.fields.phone ||
+    event.fields.ticketingUrl) && (
+    <View>
+      <View style={styles.sectionDivider} />
+      <View style={styles.content}>
+        {event.fields.accessibilityDetails && (
+          <View style={styles.detailsSection}>
+            <Text type="h2">{text.eventDetailsAccessibilityDetails}</Text>
+            <View style={styles.accessibilityDetailsItem}>
+              <Text>{event.fields.accessibilityDetails[locale]}</Text>
+            </View>
+          </View>
+        )}
+        {(event.fields.email || event.fields.phone) && (
+          <View style={styles.detailsSection}>
+            <Text type="h2">{text.eventDetailsContact}</Text>
+            {event.fields.email && (
+              <View style={styles.contactItem}>
+                <IconItem
+                  icon={<Text type="xSmall">icn</Text>}
+                  title={event.fields.email[locale]}
+                  titleType="text"
+                />
+              </View>
+            )}
+            {event.fields.phone && (
+              <View style={styles.contactItem}>
+                <IconItem
+                  icon={<Text type="xSmall">icn</Text>}
+                  title={event.fields.phone[locale]}
+                  titleType="text"
+                />
+              </View>
+            )}
+          </View>
+        )}
+        {event.fields.ticketingUrl && (
+          <View style={styles.buyButton}>
+            <Button
+              text={text.eventDetailsBuyButton}
+              url={event.fields.ticketingUrl[locale]}
+            />
+          </View>
+        )}
+      </View>
+    </View>
+  );
+
+class EventDetailsScreen extends React.Component<Props> {
```

Comment:
It's recommended to always use PureComponent for performance. Not sure when the best time to switch everyhting is though..

Reply:
Fixed, there were only a couple of places it needed adding I think